### ISSUE TITLE
rome-suppress not working on tests

### ISSUE THREAD

**Issue by KevinKelbie** [facebookexperimental/rome#154]
Description:

I've found a couple of issues while working with `rome-suppress` the first is an off-by-one line error which I've detailed how to fix below. Once the off-by-one line error is fixed the suppressions in the codebase itself work with `rome lint` but not `rome test`. When running `rome test` it returns something like

<img width="676" alt="Screenshot 2020-03-07 at 15 02 50" src="https://user-images.githubusercontent.com/19197564/76145776-be20ed80-6084-11ea-905a-36fcc7c9ed08.png">

The above error doesn't occur with `rome lint` and successfully suppresses the error.

Additionally suppressions within tests don't seem to work either.

Once we figure out why this is happening I'll be able to submit a PR for #153 

## Steps To Reproduce

1. change [this line](https://github.com/facebookexperimental/rome/blob/b8c16f9cef4553a793976e25c3bf75da0b8626fc/packages/%40romejs/diagnostics/DiagnosticsProcessor.ts#L112) to `const targetLine = suppression.loc.end.line;` to fix off-by-one line bug

2. Add a test case like which should pass but fails.

```
test('suppression test', async t => {
    const { diagnostics } = await testLint("let foo = ''; // rome-suppress lint/unusedVariables", LINT_AND_FORMAT_ENABLED_CONFIG)
    t.is(diagnostics.length, 0)
});
```

3. Try to use a suppression within the codebase and run `rome test` which should give an error similar to the screenshot above.

## Expected Results

Suppressions in the codebase and tests should work with `rome test`.

**Comment by sebmck** [comment]
The line behaviour is deliberate. It's the same way `@ts-ignore` comments work.

**Comment by sebmck** [comment]
The Rome formatter isn't going to respect comments on the same line as a node. It's extremely difficult to retain and infer the position.

**Comment by KevinKelbie** [comment]
fixed in #168

**Comment by sebmck** [comment]
Put up #174 for a more robust fix.
